configure.ac

```diff
@@ -258,6 +258,7 @@ AS_IF([test "x$enable_integration" = "xyes" -a "$HOSTOS" = "Linux"],
        ERROR_IF_NO_PROG([grep])
        ERROR_IF_NO_PROG([env])
        ERROR_IF_NO_PROG([rm])
+       ERROR_IF_NO_PROG([sha256sum])
```

Comment:
For compatibility reasons we use "shasum -a 256". This is on master. Looks like on 2.4.x we have a mix.

Reply:
This is on 2.4.X, so it's a mix right, this isn't necessarily wrong? Its just we should probably kill all sha256sum and swap them with shasum -a sha256? I would imagine that would be a follow up patch.

Reply:
> This is on 2.4.X, so it's a mix right, this isn't necessarily wrong? Its just we should probably kill all sha256sum and swap them with shasum -a sha256? I would imagine that would be a follow up patch.

Ok, I will take it as is and work from there. Thanks